.gitignore

```diff
@@ -1,5 +1,4 @@
 # Generated manuscript output files
-output/index.html
```

Comment:
Why do you want to track `index.html`?

Doesn't make sense to track unless we want to require every contributor to build manuscript locally (bad idea).